pkg/actuators/machine/actuator.go

```diff
@@ -140,7 +140,7 @@ func (a *Actuator) Create(context context.Context, machine *machinev1.Machine) e
 		return fmt.Errorf("%s: failed to patch machine: %v", machine.Name, err)
 	}
 
-	return nil
+	return a.requeueIfInstancePending(machine)
```

Comment:
we might want to pass `newestInstance` here instead as that's the source of truth